<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QStackedWidget" name="auth">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="Login_page">
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>33</number>
            </property>
            <item>
             <widget class="QLabel" name="auth_lbl_Email_Username">
              <property name="text">
               <string>Email/Username:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="auth_lbl_password">
              <property name="text">
               <string>Пароль</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLineEdit" name="auth_input_Email_Username"/>
            </item>
            <item>
             <widget class="QLineEdit" name="auth_input_password"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>26</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="auth_enter_btn">
            <property name="text">
             <string>Войти</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="auth_register_btn">
            <property name="text">
             <string>Ещё нет аккаунта</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="auth_restore_btn">
            <property name="text">
             <string>Забыл пароль</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Sign_up_page">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>33</number>
            </property>
            <item>
             <widget class="QLabel" name="reg_lbl_Username">
              <property name="text">
               <string>Username</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="reg_lbl_Email">
              <property name="text">
               <string>Email</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="reg_lbl_password">
              <property name="text">
               <string>Пароль</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="reg_lbl_code">
              <property name="text">
               <string>Код для восстановления</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget_4">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLineEdit" name="reg_input_Username"/>
            </item>
            <item>
             <widget class="QLineEdit" name="reg_input_Email"/>
            </item>
            <item>
             <widget class="QLineEdit" name="reg_input_password"/>
            </item>
            <item>
             <widget class="QLineEdit" name="reg_input_code"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>26</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="reg_back_auth_btn">
            <property name="text">
             <string>Уже есть аккаунт</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reg_registration_btn">
            <property name="text">
             <string>Зарегестрироваться</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="restore_page">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>33</number>
          </property>
          <property name="verticalSpacing">
           <number>62</number>
          </property>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLineEdit" name="restore_input_Username_Email"/>
            </item>
            <item>
             <widget class="QLineEdit" name="restore_input_code"/>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="restore_Username_Email">
              <property name="text">
               <string>Email/Username</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="restore_code">
              <property name="text">
               <string>Защитный код</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>26</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="restore_btn">
            <property name="text">
             <string>Восстановить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="auth_back_btn">
            <property name="text">
             <string>Назад</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
